src/screens/Events/User/Register/EventWaiverModal.jsx

```diff
@@ -196,10 +202,20 @@ const EventWaiverModal = ({
       )}
 
       <Styled.Row>
-        <Styled.MainButton>
+        <Styled.MainButton onClick = {togglePdf}>
           <IconSpecial width="25" height="24" viewBox="0 0 25 24" name="link" />
           Read Waiver
         </Styled.MainButton>
+        <Styled.Row style={{
+          display: showMe?"block":"none"
+        }}>
+          <iframe
+            style={{ width: "563px", height: "666px" }}
+            src={showGuardian?"/public/files/adult.pdf": "/public/files/minor.pdf"}
```

Comment:
`src={showGuardian?"/files/adult.pdf": "/files/minor.pdf"}` should make the pdf work.